crates/net/network/src/transactions/constants.rs

```diff
@@ -0,0 +1,201 @@
+    pub const DEFAULT_MAX_CAPACITY_CACHE_PENDING_FETCH: usize =
+        100 * SOFT_LIMIT_COUNT_HASHES_IN_GET_POOLED_TRANSACTIONS_REQUEST;
+
+    /// Default maximum number of hashes pending fetch to tolerate at any time. Default is half of
+    /// [`DEFAULT_MAX_CAPACITY_CACHE_PENDING_FETCH`], which defaults to 25 600 hashes, so 12 800
+    /// hashes.
+    pub const DEFAULT_MAX_COUNT_PENDING_FETCH: usize = DEFAULT_MAX_CAPACITY_CACHE_PENDING_FETCH / 2;
+
+    /* ====== LIMITED CAPACITY ON FETCH PENDING HASHES ====== */
+
+    /// Default budget for finding an idle fallback peer for any hash pending fetch, when said
+    /// search is budget constrained. Default is a sixth of [`DEFAULT_MAX_COUNT_PENDING_FETCH`],
+    /// which defaults to 12 800 hashes (the breadth of the search), divided by
+    /// [`DEFAULT_MAX_COUNT_FALLBACK_PEERS`], which defaults to 3 peers (the depth of the search),
+    /// so the 711 lru hashes in the pending hashes cache.
+    pub const DEFAULT_BUDGET_FIND_IDLE_FALLBACK_PEER: usize =
+        DEFAULT_MAX_COUNT_PENDING_FETCH / 6 / DEFAULT_MAX_COUNT_FALLBACK_PEERS as usize;
+
+    /// Default budget for finding hashes in the intersection of transactions announced by a peer
+    /// and in the cache of hashes pending fetch, when said search is budget constrained. Default
+    /// is a sixth of [`DEFAULT_MAX_COUNT_PENDING_FETCH`], which defaults to 12 800 hashes (the
+    /// breadth of the search), so 2133 lru hashes in the pending hashes cache.
+    pub const DEFAULT_BUDGET_FIND_INTERSECTION_ANNOUNCED_BY_PEER_AND_PENDING_FETCH: usize =
+        DEFAULT_MAX_COUNT_PENDING_FETCH / 6;
+
+    /* ====== SCALARS FOR USE ON FETCH PENDING HASHES ====== */
+
+    /// Default soft limit for the number of hashes in a
+    /// [`GetPooledTransactions`](reth_eth_wire::GetPooledTransactions) request, when it is filled
+    /// from hashes pending fetch. Default is half of the
+    /// [`SOFT_LIMIT_COUNT_HASHES_IN_GET_POOLED_TRANSACTIONS_REQUEST`] which by spec is 256
+    /// hashes, so 128 hashes.
+    pub const DEFAULT_SOFT_LIMIT_COUNT_HASHES_IN_GET_POOLED_TRANSACTIONS_REQUEST_ON_FETCH_PENDING_HASHES:
+    usize = SOFT_LIMIT_COUNT_HASHES_IN_GET_POOLED_TRANSACTIONS_REQUEST / 2;
+
+    /// Default soft limit for a [`PooledTransactions`](reth_eth_wire::PooledTransactions) response
+    /// when it's used as expected response in calibrating the filling of a
+    /// [`GetPooledTransactions`](reth_eth_wire::GetPooledTransactions) request, when the request
+    /// is filled from hashes pending fetch. Default is half of
+    /// [`DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE_ON_PACK_GET_POOLED_TRANSACTIONS_REQUEST`],
+    /// which defaults to 128 KiB, so 64 KiB.
+    pub const DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE_ON_FETCH_PENDING_HASHES:
+        usize = DEFAULT_SOFT_LIMIT_BYTE_SIZE_POOLED_TRANSACTIONS_RESPONSE_ON_PACK_GET_POOLED_TRANSACTIONS_REQUEST / 2;
```

Comment:
I like the documentation on these but these names are gigantic, is there any way to make this more concise?

Reply:
yeah I know they are, but not having enough info can cost a lot of time since then they are interpreted wrong. we already had this with some other constants recently. will eventually change some of these to const functions anyway, then we can asses the lengths again.